src/Support/Helpers/Report.php

```diff
@@ -10,14 +10,18 @@
      */
     function bugsnag_report(\Exception $exception)
     {
-        // Retrieve Bugsnag instance
-        $bugsnag = app('nodes.bugsnag');
-
-        // Report exception to Bugsnag
-        if ($exception instanceof \Nodes\Exceptions\Exception) {
-            $bugsnag->notifyException($exception, $exception->getMeta(), $exception->getSeverity());
-        } else {
-            $bugsnag->notifyException($exception, null, 'error');
+        try {
+            // Retrieve Bugsnag instance
+            $bugsnag = app('nodes.bugsnag');
+    
+            // Report exception to Bugsnag
+            if ($exception instanceof \Nodes\Exceptions\Exception) {
+                $bugsnag->notifyException($exception, $exception->getMeta(), $exception->getSeverity());
+            } else {
+                $bugsnag->notifyException($exception, null, 'error');
+            }   
+        } catch(\Exception $e) {
+            // Fail silent
```

Comment:
Why is the try-catch needed? I only fails of you don't have an API key right?

I'd prefer that was alerted in the service provider when registering the instance.

This way we would be alerted of the missing data instead of just silent failing :)